source/times.js

```diff
@@ -26,17 +25,18 @@ import _curry2 from './internal/_curry2.js';
  */
 var times = _curry2(function times(fn, n) {
   var len = Number(n);
-  var idx = 0;
   var list;
 
   if (len < 0 || isNaN(len)) {
     throw new RangeError('n must be a non-negative number');
   }
-  list = new Array(len);
-  while (idx < len) {
-    list[idx] = fn(idx);
-    idx += 1;
+
```

Comment:
Can you add some benchmark to explain how much benefits will gain from this optimization?

Reply:
take a look at [this](https://blog.bitsrc.io/measuring-performance-of-different-javascript-loop-types-c0e9b1d193ed) article...
I misunderstood apparently, there doesn't seem to be much difference 🤔
should i revert that while loop instead of `for` now?

Reply:
Let's go back the the `while` loop, for the reasons [already mentioned][co].  If nothing else, consistent style is always a plus!


  [co]: https://github.com/ramda/ramda/pull/2744